src/test/datascience/helpers.ts

```diff
@@ -219,6 +219,16 @@ export async function waitForInteractiveWindow(
     return notebookDocument!;
 }
 
+export async function runInteractiveWindowInput(
+    code: string,
+    interactiveWindow: InteractiveWindow,
+    newCellCount: number
+) {
+    await insertIntoInputEditor(code);
+    await vscode.commands.executeCommand('interactive.execute');
```

Comment:
Might want to port this code into other plaes where we call `interactive.execute`, else we'll end up with the same issues.